packages/common/src/openapi/schemas/Fields.json

```diff
@@ -0,0 +1,41 @@
+{
+    "type": "array",
```

Comment:
small note: it would be preferable if this was the definition for a single `field` so it can be easier to reuse and extend.